pwiz_tools/Shared/ProteowizardWrapper/MsDataFileImpl.cs

```diff
@@ -1644,8 +1667,7 @@ public override int GetHashCode()
         {
             unchecked
             {
-                int result = 0;
-                result = (result * 397) ^ (Model != null ? Model.GetHashCode() : 0);
+                int result = (Model != null ? Model.GetHashCode() : 0);

```

Comment:
This doesn't seem necessary to me. Cleaning up generated code.

Reply:
You wouldn't want to use the minimum and maximum observed IMS values, but it is better (for missing vs. zero) to use the min/max possible IMS values on a spectrum by spectrum basis. Those values would be based on the scan range for the combined spectrum. We should probably put the IM range in the <scan> element for combined spectra. Brian and I probably discussed that before but (I) forgot about it.

Reply:
Sure. That sounds good. I was just going off how I would have expected it to work when we had 2D spectra for IMS. Then, our only option would have been whether we saw any spectra in the given range. But, I agree, if it were possible to get a range from a header on the 3D spectra, that would be great.

Reply:
>> Cleaning up generated code.
It eliminates an inspection warning.